modular_skyrat/modules/aesthetics/clothing/clothing.dm

```diff
@@ -8,8 +8,8 @@
 /obj/item/clothing/suit/toggle/labcoat/rd
 	name = "research directors labcoat"
 	desc = "A Nanotrasen standard labcoat for Research Directors. It has extra layers for more protection."
-	icon = 'modular_skyrat/modules/aesthetics/clothing/items.dmi'
-	worn_icon = 'modular_skyrat/modules/aesthetics/clothing/suit.dmi'
```

Comment:
Make sure the icons are removed from these if you put them into master files. Wanna work to kill the duplicate clothing icon files we have scattered about.